run.sh

```diff
@@ -2,6 +2,12 @@
 
 RUNMODE="${1:-start}"
 
+if [ "$ENV" = "demo" ]; then
+    sed -i '/location \/status/i \        location /storybook {\n            root /docs; \n            try_files $uri $uri/ /index.html; \n        }\n' /etc/nginx/nginx.conf
```

Comment:
Can we make this line more readable? Maybe create subconfig with this line which will be included here?